extensions/amp-sidebar/0.2/test/test-amp-sidebar.js

```diff
@@ -202,6 +202,16 @@ describes.realWin(
         });
       });
 
+      it('should create styleable mask element in DOM', () => {
+        return getAmpSidebar({'stubHistory': true}).then((sidebarElement) => {
+          const impl = sidebarElement.implementation_;
+          execute(impl, 'open');
+          expect(doc.querySelectorAll('.amphtml-sidebar-mask').length).to.equal(
```

Comment:
and here 🙃
```suggestion
          expect(doc.querySelectorAll('.amp-sidebar-mask').length).to.equal(
```

Reply:
Keeping both names to prevent a larger scope.